src/pretalx/agenda/views/schedule.py

```diff
@@ -141,21 +140,7 @@ def get(self, request, event, **kwargs):
             is_visible=True
         ).prefetch_related('submission__speakers').select_related('submission', 'room').order_by('start')
         for talk in talks:
-            vevent = cal.add('vevent')
-
-            speakers = ', '.join([p.name for p in talk.submission.speakers.all()])
-            vevent.add('summary').value = f'{talk.submission.title} - {speakers}'
-            vevent.add('dtstamp').value = creation_time
-            vevent.add('location').value = str(talk.room.name)
-            vevent.add('uid').value = 'pretalx-{}-{}@{}'.format(
-                request.event.slug, talk.submission.code,
-                netloc
-            )
-
-            vevent.add('dtstart').value = talk.start.astimezone(tz)
-            vevent.add('dtend').value = talk.end.astimezone(tz)
-            vevent.add('description').value = talk.submission.abstract or ""
-            vevent.add('url').value = talk.submission.urls.public.full(scheme='https')
+            talk.build_ical(cal, creation_time=creation_time, netloc=netloc)
```

Comment:
`creation_time` already defaults to `datetime.now(pytz.utc)`. Actually, it is seemingly never set to something other than that. In case there isn't some hidden meaning I don't get: think about removing the parameter or at least don't pass it over here.

Reply:
When generating a lot of talks for *one* ical, I want a consistent creation time (and only call `now()` once, no reason to have `n` method calls when you can have just one.